feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SearchFragment.kt

```diff
@@ -46,16 +48,22 @@ import javax.inject.Provider
 class SearchFragment : DaggerFragment() {
     private lateinit var binding: FragmentSearchBinding
 
-    @Inject lateinit var searchActionCreator: SearchActionCreator
-    @Inject lateinit var speechSessionItemFactory: SpeechSessionItem.Factory
-    @Inject lateinit var serviceSessionItemFactory: ServiceSessionItem.Factory
-    @Inject lateinit var sessionContentsStore: SessionContentsStore
+    @Inject
```

Comment:
I would like to use one line by annotation format rule 👀 
https://kotlinlang.org/docs/reference/coding-conventions.html#annotation-formatting
https://github.com/yole/kotlin-style-guide/issues/4

Reply:
I fixed it 🙇